src/Azure.Functions.Cli/Actions/LocalActions/InitAction.cs

```diff
@@ -404,6 +404,17 @@ private static async Task WriteLocalSettingsJson(WorkerRuntime workerRuntime, Pr
 
         private static async Task WriteDockerfile(WorkerRuntime workerRuntime, string language, string targetFramework, bool csx)
         {
+            if (WorkerRuntimeLanguageHelper.IsDotnet(workerRuntime) && string.IsNullOrEmpty(targetFramework))
+            {
+                var functionAppRoot = ScriptHostHelpers.GetFunctionAppRootDirectory(Environment.CurrentDirectory);
+                string projectFilePath = ProjectHelpers.FindProjectFile(functionAppRoot);
```

Comment:
Removed the code to find project file path here since the helper does that.

Reply:
Hi @kshyju done the final changes.